### DISCUSSION TITLE
Swagger

### DISCUSSION THREAD

**Discussion by EhsanHabib** [jasontaylordev/CleanArchitecture#531] [Q&A]
I have added a new controller in the controllers folder but it is not appearing in the swagger api.

**Comment by neozhu** [comment]
can you share your code

**Reply by EhsanHabib** [reply]
![image](https://user-images.githubusercontent.com/15071654/146898112-4ee0277d-d543-46b8-a585-a4a787927969.png)

**Reply by neozhu** [reply]
Look good to me, 
try delete bin folder and restart

**Reply by EhsanHabib** [reply]
didnot work

**Reply by ramax495** [reply]
Try to Run project in Debug mode.
Command in WebUI.csproj executes only on Debug mode.

`<Target Name="NSwag" AfterTargets="PostBuildEvent" Condition=" '$(Configuration)' == 'Debug' ">`
